<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Boolean_set_operations_2</class>
 <widget class="QMainWindow" name="Boolean_set_operations_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Boolean Set Operations and Minkowksi Sum</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Boolean_operations_2.qrc">
    <normaloff>:/cgal/icons/resources/cgal_logo.xpm</normaloff>:/cgal/icons/resources/cgal_logo.xpm</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::NoAnchor</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>978</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionErase"/>
    <addaction name="actionEraseAll"/>
    <addaction name="actionDuplicate"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionQuit"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionShowHide"/>
    <addaction name="actionSelectAllItems"/>
    <addaction name="actionPreferences"/>
    <addaction name="actionColorItems"/>
   </widget>
   <widget class="QMenu" name="menuOperations">
    <property name="title">
     <string>&amp;Operations</string>
    </property>
    <addaction name="actionInsertLinear"/>
    <addaction name="actionInsertCircular"/>
    <addaction name="actionComplement"/>
    <addaction name="actionintersection"/>
    <addaction name="actionunion"/>
    <addaction name="actionDifference"/>
    <addaction name="actionSymmetric_Difference"/>
    <addaction name="actionMinkowski_Sum"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionRecenter"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuOperations"/>
   <addaction name="menuView"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="sceneDockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="windowTitle">
    <string>Geometric Objects</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContent">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="addButton">
           <property name="text">
            <string>+</string>
           </property>
           <property name="icon">
            <iconset resource="Boolean_operations_2.qrc">
             <normaloff>:/cgal/icons/plus</normaloff>:/cgal/icons/plus</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="removeButton">
           <property name="text">
            <string>-</string>
           </property>
           <property name="icon">
            <iconset resource="Boolean_operations_2.qrc">
             <normaloff>:/cgal/icons/minus</normaloff>:/cgal/icons/minus</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="duplicateButton">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="Boolean_operations_2.qrc">
             <normaloff>:/cgal/icons/duplicate</normaloff>:/cgal/icons/duplicate</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="upButton">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="Boolean_operations_2.qrc">
             <normaloff>:/cgal/icons/resources/up.png</normaloff>:/cgal/icons/resources/up.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="downButton">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="Boolean_operations_2.qrc">
             <normaloff>:/cgal/icons/resources/down.png</normaloff>:/cgal/icons/resources/down.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="searchEdit">
           <property name="placeholderText">
            <string>Quick filter... &lt;Alt+Q&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="sceneView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="consoleDockWidget">
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::LeftDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Console</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTextEdit" name="consoleTextEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="infoDockWidget">
   <property name="windowTitle">
    <string>Infos</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>541</width>
            <height>173</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="infoLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>1</verstretch>
              </sizepolicy>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::DefaultContextMenu</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
             <property name="textInteractionFlags">
              <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="displayLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="Boolean_operations_2.qrc">
     <normaloff>:/cgal/icons/plus</normaloff>:/cgal/icons/plus</iconset>
   </property>
   <property name="text">
    <string>&amp;Load curve</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionErase">
   <property name="icon">
    <iconset resource="Boolean_operations_2.qrc">
     <normaloff>:/cgal/icons/minus</normaloff>:/cgal/icons/minus</iconset>
   </property>
   <property name="text">
    <string>&amp;Erase curve</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionDuplicate">
   <property name="icon">
    <iconset resource="Boolean_operations_2.qrc">
     <normaloff>:/cgal/icons/duplicate</normaloff>:/cgal/icons/duplicate</iconset>
   </property>
   <property name="text">
    <string>&amp;Duplicate curve</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionAntiAliasing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Antialiasing</string>
   </property>
  </action>
  <action name="dummyAction">
   <property name="text">
    <string>n/a</string>
   </property>
  </action>
  <action name="actionEraseAll">
   <property name="text">
    <string>&amp;Erase All curves</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save &amp;as...</string>
   </property>
  </action>
  <action name="actionShowHide">
   <property name="text">
    <string>Show/Hide</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
  </action>
  <action name="actionDrawTwoSides">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Draw &amp;Two Sides</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionRecenter">
   <property name="text">
    <string>Re&amp;center Scene</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSetBackgroundColor">
   <property name="text">
    <string>Change &amp;Background Color...</string>
   </property>
  </action>
  <action name="actionLookAt">
   <property name="text">
    <string>&amp;Look at...</string>
   </property>
  </action>
  <action name="actionSaveSnapshot">
   <property name="text">
    <string>Save Snapshot</string>
   </property>
  </action>
  <action name="actionDumpCamera">
   <property name="text">
    <string>&amp;Dump Camera Coordinates</string>
   </property>
  </action>
  <action name="actionCopyCamera">
   <property name="text">
    <string>&amp;Copy Camera</string>
   </property>
  </action>
  <action name="actionPasteCamera">
   <property name="text">
    <string>&amp;Paste Camera</string>
   </property>
  </action>
  <action name="actionSelectAllItems">
   <property name="text">
    <string>Select All Items</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionLoadScript">
   <property name="text">
    <string>Load &amp;Script</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
  <action name="actionLoadPlugin">
   <property name="text">
    <string>Load Plugin</string>
   </property>
  </action>
  <action name="actionQuickCameraMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Quick Camera Mode</string>
   </property>
  </action>
  <action name="actionMaxTextItemsDisplayed">
   <property name="text">
    <string>Set Maximum  Text Items Displayed :</string>
   </property>
  </action>
  <action name="actionSwitchProjection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Orthographic Projection</string>
   </property>
  </action>
  <action name="actionColorItems">
   <property name="text">
    <string>Set Different Colors for Selected Items</string>
   </property>
  </action>
  <action name="actionPolyhedron_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Polyhedron Mode</string>
   </property>
   <property name="toolTip">
    <string>In Polyhedron Mode, the default type for facegraphs is Polyhedron, by opposition to Surface_mesh.</string>
   </property>
  </action>
  <action name="actionInsertLinear">
   <property name="icon">
    <iconset resource="Boolean_operations_2.qrc">
     <normaloff>:/cgal/icons/resources/insert_bezier.bmp</normaloff>:/cgal/icons/resources/insert_bezier.bmp</iconset>
   </property>
   <property name="text">
    <string>Draw Linear polygon</string>
   </property>
  </action>
  <action name="actionInsertCircular">
   <property name="icon">
    <iconset resource="Boolean_operations_2.qrc">
     <normaloff>:/cgal/icons/resources/insert_polygon.bmp</normaloff>:/cgal/icons/resources/insert_polygon.bmp</iconset>
   </property>
   <property name="text">
    <string>Draw Circular polygon</string>
   </property>
  </action>
  <action name="actionComplement">
   <property name="icon">
    <iconset resource="Boolean_operations_2.qrc">
     <normaloff>:/cgal/icons/resources/comp_P.bmp</normaloff>:/cgal/icons/resources/comp_P.bmp</iconset>
   </property>
   <property name="text">
    <string>Complement</string>
   </property>
  </action>
  <action name="actionintersection">
   <property name="icon">
    <iconset resource="Boolean_operations_2.qrc">
     <normaloff>:/cgal/icons/resources/intersection.bmp</normaloff>:/cgal/icons/resources/intersection.bmp</iconset>
   </property>
   <property name="text">
    <string>Intersection</string>
   </property>
  </action>
  <action name="actionunion">
   <property name="icon">
    <iconset resource="Boolean_operations_2.qrc">
     <normaloff>:/cgal/icons/resources/union.bmp</normaloff>:/cgal/icons/resources/union.bmp</iconset>
   </property>
   <property name="text">
    <string>Union</string>
   </property>
  </action>
  <action name="actionDifference">
   <property name="icon">
    <iconset resource="Boolean_operations_2.qrc">
     <normaloff>:/cgal/icons/resources/diff_PQ.bmp</normaloff>:/cgal/icons/resources/diff_PQ.bmp</iconset>
   </property>
   <property name="text">
    <string>Difference</string>
   </property>
  </action>
  <action name="actionSymmetric_Difference">
   <property name="icon">
    <iconset resource="Boolean_operations_2.qrc">
     <normaloff>:/cgal/icons/resources/symm_diff.bmp</normaloff>:/cgal/icons/resources/symm_diff.bmp</iconset>
   </property>
   <property name="text">
    <string>Symmetric Difference</string>
   </property>
  </action>
  <action name="actionMinkowski_Sum">
   <property name="icon">
    <iconset resource="Boolean_operations_2.qrc">
     <normaloff>:/cgal/icons/resources/mink_sum.bmp</normaloff>:/cgal/icons/resources/mink_sum.bmp</iconset>
   </property>
   <property name="text">
    <string>Minkowski Sum</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionDraw_Polygon">
   <property name="text">
    <string>Draw Polygon</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionFirst_Polygon">
   <property name="text">
    <string>First Polygon</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Boolean_operations_2.qrc"/>
 </resources>
 <connections/>
</ui>
